src/main/resources/swagger/rawls.yaml

```diff
@@ -789,10 +792,10 @@ paths:
           schema:
             $ref: '#/definitions/ErrorReport'
         '502':
-          description: Trouble talking to Cromwell
+          description: Trouble talking to Agora
```

Comment:
We actually eat any errors from Cromwell during submissions; they get turned into WorkflowFailures rather than bad return codes. `withMethodInputs` is what returns 502, which is why I documented it like this.